kie-bom/pom.xml

```diff
@@ -157,6 +157,19 @@
         <classifier>sources</classifier>
       </dependency>
 
+      <!-- KIE Security -->
+      <dependency>
+        <groupId>org.drools</groupId>
```

Comment:
if that won't be an issues to have it org.kie in droolsjbpm-integration repository....